test/basic-auth.js

```diff
@@ -109,6 +109,29 @@ describe('auth(req)', function () {
       assert.strictEqual(creds.pass, 'pass:word')
     })
   })
+
+  describe('with different case', function () {
+    it('should return .name and .pass', function () {
+      var req = request('Basic Zm9vOmJhcg==')
+      var creds = auth(req)
+      assert.strictEqual(creds.name, 'foo')
+      assert.strictEqual(creds.pass, 'bar')
+    })
+
+    it('should return .name and .pass', function () {
+      var req = request('BASIC Zm9vOmJhcg==')
+      var creds = auth(req)
+      assert.strictEqual(creds.name, 'foo')
+      assert.strictEqual(creds.pass, 'bar')
+    })
+
+    it('should return .name and .pass', function () {
```

Comment:
When the test suite is run, all these tests appear with exactly the same name & path, making a failure when it is summed up at the bottom had to tell which case failed. Can you either (a) separate these into different `describe` blocks or (b) change the names for the `it` to be different for each within the same `describe` block?

Reply:
Hey @dougwilson, I changed each test case (`it`) appending the word is testing, hopefully this is clearer.

Ref: https://github.com/jshttp/basic-auth/pull/41/commits/fb21123a2dedea7fe2560531e38d3b24ec719f10